.gitmodules

```diff
@@ -43,6 +34,33 @@
 [submodule "1-trackers/golang-tracker"]
 	path = 1-trackers/golang-tracker
 	url = https://github.com/snowplow/snowplow-golang-tracker.git
-[submodule "4-storage/kinesis-s3"]
-	path = 4-storage/kinesis-s3
-	url = https://github.com/snowplow/kinesis-s3.git
+[submodule "4-storage/s3-loader"]
+	path = 4-storage/s3-loader
+	url = https://github.com/snowplow/snowplow-s3-loader.git
+[submodule "3-enrich/enrich"]
+	path = 3-enrich/enrich
+	url = git@github.com:snowplow/enrich.git
+[submodule "2-collectors/stream-collector"]
+	path = 2-collectors/stream-collector
+	url = git@github.com:snowplow/stream-collector.git
+[submodule "3-enrich/eer"]
+	path = 3-enrich/emr-etl-runner
+	url = git@github.com:snowplow/emr-etl-runner.git
+[submodule "4-storage/snowplow-rdb-loader"]
+	path = 4-storage/snowplow-rdb-loader
+	url = git@github.com:snowplow/snowplow-rdb-loader.git
+[submodule "4-storage/snowplow-bigquery-loader"]
+	path = 4-storage/snowplow-bigquery-loader
+	url = git@github.com:snowplow-incubator/snowplow-bigquery-loader.git
+[submodule "4-storage/snowplow-snowflake-loader"]
+	path = 4-storage/snowplow-snowflake-loader
+	url = git@github.com:snowplow-incubator/snowplow-snowflake-loader.git
+[submodule "4-storage/gcs-loader"]
+	path = 4-storage/gcs-loader
+	url = git@github.com:snowplow-incubator/snowplow-google-cloud-storage-loader.git
+[submodule "4-storage/es-loader"]
+	path = 4-storage/es-loader
+	url = git@github.com:snowplow/snowplow-elasticsearch-loader.git
+[submodule "4-storage/postgres"]
```

Comment:
Do we need submodules for the analytics sdks too?

Reply:
You're right I will add them, as well as Mini and Micro